### PR TITLE
Fixes #2322 - rename `fix` to `lint:fix`in pkg.json

### LINKED ISSUE #2322: change `fix` to `lint:fix` in package.json
Running into this pretty much every time, so I'm changing this. :D

### PR DESCRIPTION
**Pull request by zoepage**
r? @miketaylr

### CODE REVIEW

File: docs/pr-coding-guidelines.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ In order to avoid errors during a Pull Request, `npm run lint` will be executed
 `npm run lint` checks all relevant JavaScript and CSS files and displays, if something needs to be fixed.
 
 If you get an error displayed, there are two ways to fix it.
-1. You can run `npm run fix` automatically (only for JavaScript), which is great for small issues like missing spaces or lines in various files.
+1. You can run `npm run lint:fix` automatically (only for JavaScript), which is great for small issues like missing spaces or lines in various files.
```
Comment (miketaylr) [inline docs/pr-coding-guidelines.md]: The docs say this is only for JS, but `lint:fix` does `lint:fix:css`. Are the docs out of date?

**Comment by miketaylr** [comment]
(also, small typo in the commit message: upadte docs)

File: docs/pr-coding-guidelines.md (hunk 1 above)
Comment (zoepage) [reply docs/pr-coding-guidelines.md]: They ARE! Thanks for pointing that out :)

**Pull request merged**